api/app/controllers/mno_enterprise/jpi/v1/admin/organizations_controller.rb

```diff
@@ -100,12 +100,13 @@ def user_params
     def create_unconfirmed_user(user_params)
       user = MnoEnterprise::User.new(user_params)
       user.skip_confirmation_notification!
-      user.save
+      user.password = Devise.friendly_token
+      user.save!
 
-      # Reset the confirmation field so we can track when the invite is send - #confirmation_sent_at is when the confirmation_token was generated (not sent)
-      # Not ideal as we do 2 saves, and the previous save trigger a call to the backend to validate the token uniqueness
-      user.attributes = {confirmation_sent_at: nil, confirmation_token: nil}
-      user.save
+      # # Reset the confirmation field so we can track when the invite is send - #confirmation_sent_at is when the confirmation_token was generated (not sent)
+      # # Not ideal as we do 2 saves, and the previous save trigger a call to the backend to validate the token uniqueness
+      # user.attributes = {confirmation_sent_at: nil, confirmation_token: nil}
```

Comment:
Well spotted, I was doing some test, and I did not check if it was the same behavior.